apps/console/src/features/applications/data/application-templates/templates/single-page-application/create-wizard-help.tsx

```diff
@@ -56,7 +56,10 @@ const SinglePageApplicationCreateWizardHelp: FunctionComponent<SinglePageApplica
                     After user login/logout, the user is redirected to a web page within your application
                     that receives the post-authentication authorization code.
                 </p>
-                <p>Add the list of possible redirect URIs here. You can specify multiple valid URIs.</p>
+                <p>
+                    Specify the list of possible redirect URIs here, so that WSO2 Identity Server
```

Comment:
@yasinmiran Please refactor `WSO2 Identity Server` part with the value from the config.

```suggestion
                    Specify the list of possible redirect URIs here, so that WSO2 Identity Server
```

Reply:
+1 This has is addressed in commit e37168e4b52b7f4f745b5d4907bfdef15be16969